docs/docs/Components/components-vector-stores.md

```diff
@@ -78,6 +80,54 @@ For an example of using the **Astra DB Vector Store** component with an embeddin
 
 For more information, see the [Astra DB Serverless documentation](https://docs.datastax.com/en/astra-db-serverless/databases/embedding-generation.html).
 
+### Hybrid search
+
+The **Astra DB** component includes **hybrid search** enabled by default.
+
+The component fields related to hybrid search are **Search Query**, **Lexical Terms**, and **Reranker**.
+
+* **Search Query** finds results by vector similarity.
+* **Lexical Terms** is a comma-separated list of keywords.
+* **Reranker** is the re-ranker model used in the hybrid search.
+The default model is `nvidia/llama-3.2-nv.reranker`.
+
+**Hybrid search** passes a vector similarity search and a lexical search to the [find and rerank Data API endpoint](https://docs.datastax.com/en/astra-db-serverless/api-reference/document-methods/find-and-rerank.html), where a reranker combines and sorts the results based on the vector search's similarity to the lexical terms.
```

Comment:
```suggestion
(https://docs.datastax.com/en/astra-db-serverless/databases/hybrid-search.html[**Hybrid search**] performs a vector similarity search and a lexical search, compares the results of both searches, and then returns the most relevant results overall.
```


You can update the link because the hybrid-search page will link to the specific command.